src/Exception/Renderer/InlineRenderer.php

```diff
@@ -25,22 +25,22 @@
 class InlineRenderer implements RendererInterface
 {
     /**
-     * @var EngineInterface
+     * @var Environment
      */
-    protected $templating;
+    protected $twig;
```

Comment:
Should be private

Reply:
Fixed, `$twig` property is private in `InlineRenderer` and `InlineDebugRenderer`